string node_name
float32 timeout
---
bool success
